CMakeLists.txt

```diff
@@ -18,6 +20,7 @@ if(EXISTS ${CMAKE_CURRENT_SOURCE_DIR}/evm2wasm/cmake/HunterGate.cmake)
 endif()
 
 project(hera)
+set(PROJECT_VERSION 0.1.0)
```

Comment:
I feel this should be done its own PR, adding proper build info. Is this line needed for this PR? If not, please remove. Otherwise please create a buildinfo PR first.

Reply:
This must be here, because CMake needs the version specified for making the package. I just put any number there. Can be also `0.0.0`.